src/main/resources/data.sql

```diff
@@ -0,0 +1,7 @@
+
+
+
+INSERT INTO User VALUES(12,'danmargreen1','Danmar1','Green1','$2a$04$5j.ZgXSEc.v3w.s7yTbiRelZ73.AJfVd4l4Z.7soePVPLb6ZgTQ6O','5955 Skyline Dr, West Linn, OR, 97068','1996-01-17', 'danmargreen0@gmail.com','(503)-833-2453', NOW(), NOW());
+INSERT INTO User VALUES(14,'danmargreen2','Danmar2','Green2','$2a$10$SkyNcaKZ1ogQTMeSL9xIruwyqUyzsYd.pwR0twS/khEpS8hkzm6CC','5955 Skyline Dr, West Linn, OR, 97068','1996-01-17', 'danmargreen1@gmail.com','(503)-833-2453', NOW(), NOW());
+INSERT INTO Auth_User_Group VALUES(12,'danmargreen1', 'ROLE_ADMIN');
+INSERT INTO Auth_User_Group VALUES(14,'danmargreen2', 'ROLE_USER');
```

Comment:
Don't ask me why ( it has been bugging me as a developer :-) ) There needs to be a newline in the end of each file.